src/main/java/com/epam/preprod/sirenko/UniqueObjectsList.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.preprod.sirenko;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class UniqueObjectsList extends ArrayList<Object> {
```

Comment:
Can we use ArrayList `<E>`?

Reply:
When use <E> I get this: 'add(Object)' in 'com.epam.preprod.sirenko.UniqueObjectsList' clashes with 'add(E)' in 'java.util.ArrayList'; both methods have same erasure, yet neither overrides the other
Only using <Object> works fine